src/bank.rs

```diff
@@ -296,6 +317,13 @@ mod test {
         let raw = bank.query(&api, &store, &querier, &block, req).unwrap();
         let res: BalanceResponse = from_slice(&raw).unwrap();
         assert_eq!(res.amount, coin(0, "eth"));
+
+        let req = BankQuery::Supply {
+            denom: "eth".into(),
+        };
+        let raw = bank.query(&api, &store, &querier, &block, req).unwrap();
+        let res: SupplyResponse = from_slice(&raw).unwrap();
+        assert_eq!(res.amount, coin(100, "eth"));
```

Comment:
Nice test. Another case where the amount is owned by multiple accounts would be nice.

Reply:
Great! Will add.

Reply:
Done!